<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View</class>
 <widget class="QMainWindow" name="View">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>521</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1500</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Stratego</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1000</width>
     <height>1000</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1600</width>
     <height>1050</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>831</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="start_Menu">
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>481</width>
        <height>411</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="menuBox">
       <property name="spacing">
        <number>40</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>70</number>
       </property>
       <item>
        <widget class="QLabel" name="difficulty">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>450</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>450</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Choisissez la difficultée souhaitée :</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QComboBox" name="menuChoice">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Amateur</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Normal</string>
          </property>
         </item>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="fillingMod">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Quel mode de remplissage des pions souhaitez-vous ? </string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="topMargin">
          <number>50</number>
         </property>
         <item>
          <widget class="QPushButton" name="interactive_start">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Mode Interactif</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="file_start">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Chargement Fichiers</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="gameTitle">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>0</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">h1 {
	font: 900 9pt &quot;Arial Black&quot;;
	font: 24pt &quot;Segoe UI&quot;;
  text-align: center;
  color: white;
  text-transform: uppercase;
  padding: 1px;
  font-family: &quot;Raleway&quot;, cursive;
  font-weight: 100;
  position: relative;
  width: 100%;
  background: linear-gradient(to right, black, #eee, black);
  white-space: nowrap;
}
h1::before {
  content: &quot;&quot;;
  position: absolute;
  left: 50%;
  top: -50px;
  width: 600px;
  margin-left: -300px;
  margin-top: -220px;
  height: 600px;
  background: radial-gradient(ellipse closest-side, #444, black);
  z-index: -1;
}
h1 a {
  background: black;
  display: block;
  padding: 20px;
  text-decoration: none;
  letter-spacing: 30px;
  color: white;
}

body {
  background: black;
  display: grid;
  place-items: center;
  height: 100vh;
  margin: 0;
  overflow: hidden;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="text">
       <string>STRATEGO</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="game">
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>441</width>
        <height>431</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="game_G">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="horizontalSpacing">
        <number>6</number>
       </property>
       <property name="verticalSpacing">
        <number>10</number>
       </property>
      </layout>
     </widget>
     <widget class="QLabel" name="titleInfos">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>30</y>
        <width>311</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Couleur</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="infosLiveLabel">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>140</y>
        <width>301</width>
        <height>181</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Infos en live</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="lastKilledRed">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>400</y>
        <width>91</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="lastKilledBlue">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>400</y>
        <width>93</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelKilledPiece">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>330</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Dernière pièce capturée</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>431</width>
        <height>421</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(../Stratego_FinalV/resources/war.png)</string>
      </property>
     </widget>
     <zorder>label_2</zorder>
     <zorder>gridLayoutWidget</zorder>
     <zorder>titleInfos</zorder>
     <zorder>infosLiveLabel</zorder>
     <zorder>lastKilledRed</zorder>
     <zorder>lastKilledBlue</zorder>
     <zorder>labelKilledPiece</zorder>
    </widget>
    <widget class="QWidget" name="interactive_P">
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>431</width>
        <height>361</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="interactiv_put">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>10</y>
        <width>321</width>
        <height>451</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="interactiv_select"/>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>828</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionManuel"/>
   </widget>
   <widget class="QMenu" name="menuJeu">
    <property name="title">
     <string>Jeu</string>
    </property>
    <addaction name="actionNouvelle_Partie"/>
   </widget>
   <addaction name="menuHelp"/>
   <addaction name="menuJeu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionNouvelle_Partie">
   <property name="text">
    <string>Nouvelle Partie</string>
   </property>
  </action>
  <action name="actionManuel">
   <property name="text">
    <string>Manuel</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
